python/generate_classification.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Not sure where this script should belong

Reply:
create an `integration_test` dir under classification tests? Here is not a good place ;-)

Reply:
can we rename the whole file to something more descriptive? Perhaps `run_classifier.py` or `train_and_predict.py` or something else? the `generate` usually refers to generating classification data which is not what we are doing here.

Reply:
Oops, I missed your naming recommendation... I went for "generate_model.sh"... I could do train_model.sh if this suits you best.